admin/css/views/stats.css

```
@@ -906,6 +1278,40 @@
+@media (max-width: 720px) {
+    .stats-peak-pair {
+        grid-template-columns: 1fr;
+    }
+
+    .stats-peak-pair,
+    .stats-note-slot,
+    .max-mag-card,
+    .snet-max-card,
+    .stats-note-card {
+        --stats-bottom-card-height: 180px;
+        height: var(--stats-bottom-card-height);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)